packages/react-sdk/src/useSpaceList.ts

```diff
@@ -0,0 +1,11 @@
+'use client'
+
+import { useRiver } from './useRiver'
+
+/*
+ * Returns a list with all space ids.
+ */
+export const useSpaceList = () => {
+    const { data, ...rest } = useRiver((s) => s.spaces)
+    return { spaceIds: data.spaceIds, ...rest }
+}
```

Comment:
This is confusing to me. I don’t like the name, and it looks like it’s just returning the same object

Reply:
maybe `useAllSpaces`?

Reply:
Is this supposed to be the spaces that the user is joined? Then I suggest we pick one of the following naming conventions: “useMySpaces”, “useJoinedSpaces”, or “useUserSpaces”

there are a bunch of things where this applies, we should pick something and stick to it. I personally like the “my” moniker, since it applies to lots of things like my settings, my channels, my device keys, my inbox, etc